core/lib/types/src/l1/mod.rs

```diff
@@ -274,7 +274,7 @@ impl From<L1Tx> for abi::NewPriorityRequest {
             transaction: abi::L2CanonicalTransaction {
                 tx_type: PRIORITY_OPERATION_L2_TX_TYPE.into(),
                 from: address_to_u256(&t.common_data.sender),
-                to: address_to_u256(&t.execute.contract_address),
+                to: address_to_u256(&t.execute.contract_address.unwrap_or_default()),
```

Comment:
L1 transactions come from `L2TransactionRequest`s, which have to contain an address, so I don't think `contract_address` is nullable.

I imagine making it non-nullable through the type system would involve making a new `Execute` struct specific for L1 transactions where the `contract_address` field is not an `Option` and make L1Tx contain it instead of the regular `Execute`. Is that reasonable or do you think it's better to unwrap/use try_from to return an error instead?

Reply:
> L1 transactions come from L2TransactionRequests, which have to contain an address, so I don't think contract_address is nullable.

Yes, that's the current state, I was asking more about future plans. I didn't expect a definite answer but I wanted to raise this topic

> Is that reasonable or do you think it's better to unwrap/use try_from to return an error instead?

I think it's reasonable long-term but not as a part of this PR. So let's unwrap/return error for now but keep it in mind, cc @0xVolosnikov